README.md

```
@@ -98,6 +100,54 @@ If the documentation doesn't cover what you need, search the [many questions on
+This template supports running load tests on AWS to evaluate the performance of your application under various conditions. You can automate this process using a custom bash script that provisions an EC2 instance, attaches an IAM role, creates an S3 bucket for storing the results, and executes the load tests.
+
+### Steps for Running AWS Load Tests
+
+#### 1. **Configure AWS CLI**:
+   Before you can interact with AWS, you'll need to [configure the AWS CLI](https://docs.aws.amazon.com/cli/v1/userguide/cli-chap-configure.html) with your credentials. 
+   Run the following command and provide your AWS Access Key, Secret Access Key. Also, 
+
+#### 2. Configure Environment Variables in config.sh
+You can configure the following environment variables in the [tests/Load/config.sh](https://github.com/VilnaCRM-Org/php-service-template/blob/main/tests/Load/config.sh) file, depending on your project needs:
+
+* **`REGION`**: Defines the AWS region where the EC2 instance will be launched (e.g., `us-east-1`)
```

Markdownlint

Expected: dash; Actual: asterisk
Unordered list style

(MD004, ul-style)